bip-0155.mediawiki

```diff
@@ -78,8 +78,8 @@ This means that the message contains a serialized <code>std::vector</code> of th
 
 One message can contain up to 1,000 addresses. Clients SHOULD reject messages with more addresses.
 
-Field <code>addr</code> has a variable length, with a maximum of 32 bytes (256 bits). Clients SHOULD reject
-longer addresses.
+Field <code>addr</code> has a variable length, with a maximum of 64 bytes (512 bits).
+Clients SHOULD reject messages with longer addresses, even if they are for an unknown network ID (old client seeing a message from a newer client that supports a new network ID).
```

Comment:
Nit: Could perhaps be reworded, may be confused if the parenthetical "(old client seeing a message from a newer client that supports a new network ID)" refers to "unknown network ID" or "messages with longer addresses"

Perhaps: Clients SHOULD reject messages with longer addresses, irrespective of the network ID.

Reply:
Updated to your proposal, much shorter and clear.